server/internal/orchestrator/swarm/mcp_config_test.go

```
@@ -0,0 +1,595 @@
+package swarm
+
+import (
+	"testing"
+
+	"github.com/pgEdge/control-plane/server/internal/database"
+	"gopkg.in/yaml.v3"
+)
+
+func strPtr(s string) *string       { return &s }
+func float64Ptr(f float64) *float64 { return &f }
```

golangci-lint (2.5.0)

[error] func float64Ptr is unused

(unused)